### PR TITLE
Add test for package integrity

### PR DESCRIPTION
**Pull request by PicoCentauri**
Since we had some twine issues for the 0.2.0 release and add a test for the package integrity.

Contributor (creator of PR) checklist
-------------------------------------
 - [ ] Tests updated (for new features and bugfixes)?
 - [ ] Documentation updated (for new features)?
 - [ ] Issue referenced (for PRs that solve an issue)?

For Reviewer
------------
 - [x] CHANGELOG updated if important change?

----
:books: Documentation preview :books:: https://scikit-matter--215.org.readthedocs.build/en/215/

### CODE REVIEW

File: developer/clean_dist_check.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```
Comment (agoscinski) [inline developer/clean_dist_check.py:1]: not sure, isn't much less code if we do in the `tox.ini`
```
allowlist_externals = unlink
commands_pre = 
    unlink {toxinidir}/dist/*.whl`
    unlink {toxinidir}/dist/*.tar.gz
```


Also it is a cleanup for `tox -e build` right?

Comment (PicoCentauri) [reply developer/clean_dist_check.py:1]: yes you are right

Comment (agoscinski) [reply developer/clean_dist_check.py:1]:
```
allowlist_externals = bash
commands_pre =
    bash -c "if [ -e {toxinidir}/dist/*tar.gz ]; then unlink {toxinidir}/dist/*.whl; fi"
    bash -c "if [ -e {toxinidir}/dist/*tar.gz ]; then unlink {toxinidir}/dist/*.tar.gz; fi"

```
I don't mind to have a script folder for development tools and to have it in Python, but in this case my impression is that it is not so complicated to do it in bash to include it in the tox.

Comment (PicoCentauri) [reply developer/clean_dist_check.py:1]: Yes would be fine for me. I check the code.

**Review by agoscinski — APPROVED** [review]
Thanks Philip for the work!

**Pull request merged**